catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -625,4 +648,12 @@ protected String parseTextFrom(String xmlData) {
 
         return newAttributeDescriptors;
     }
+
+    String getSortKey(String field) {
```

Comment:
I don't believe so, looks like an oversight during an automatic refactoring, I'll fix it.